cmd/nerdctl/stats_linux_test.go

```diff
@@ -32,9 +33,13 @@ func TestStats(t *testing.T) {
 		t.Skip("test skipped for rootless containers on cgroup v1")
 	}
 	testContainerName := testutil.Identifier(t)
+	randomTestContainerName := fmt.Sprintf("%s-random", testContainerName)
 
 	base := testutil.NewBase(t)
 	defer base.Cmd("rm", "-f", testContainerName).Run()
+	defer base.Cmd("rm", "-f", randomTestContainerName).Run()
+	base.Cmd("run", "-d", "--name", randomTestContainerName, testutil.NginxAlpineImage).AssertOK()
+	base.Cmd("kill", randomTestContainerName).AssertOK()
```

Comment:
no need to kill this will automatically exit https://github.com/containerd/nerdctl/pull/1416/files#diff-c1ea01872000c1855f6cfc8180843ee41c96d0f5d54cc479681a96956caf6a58R41